modules/nw-enabling-hsts.adoc

```diff
@@ -3,54 +3,19 @@
 // * networking/configuring-routing.adoc
 
 [id="nw-enabling-hsts_{context}"]
-= Enabling HTTP strict transport security
+= HTTP strict transport security
 
-HTTP Strict Transport Security (HSTS) policy is a security enhancement, which
-ensures that only HTTPS traffic is allowed on the host. Any HTTP requests are
-dropped by default. This is useful for ensuring secure interactions with
-websites, or to offer a secure application for the user's benefit.
+HTTP Strict Transport Security (HSTS) policy is a security enhancement, which ensures that only HTTPS traffic is allowed on the host. Any HTTP requests are dropped by default. This is useful for ensuring secure interactions with websites, or to offer a secure application for the user's benefit.
 
-When HSTS is enabled, HSTS adds a Strict Transport Security header to HTTPS
-responses from the site. You can use the `insecureEdgeTerminationPolicy` value
-in a route to redirect to send HTTP to HTTPS. However, when HSTS is enabled, the
-client changes all requests from the HTTP URL to HTTPS before the request is
-sent, eliminating the need for a redirect. This is not required to be supported
-by the client, and can be disabled by setting `max-age=0`.
+When HSTS is enforced, HSTS adds a Strict Transport Security header to HTTPS responses from the site. You can use the `insecureEdgeTerminationPolicy` value in a route to redirect to send HTTP to HTTPS. However, when HSTS is enforced, the client changes all requests from the HTTP URL to HTTPS before the request is sent, eliminating the need for a redirect. This is not required to be supported by the client, and can be disabled by setting `max-age=0`.
+
+Cluster administrators can:
+
+* Enforce HSTS per-route
+* Disable HSTS per-route
+* Enforce HSTS per-domain
 
 [IMPORTANT]
 ====
-HSTS works only with secure routes (either edge terminated or re-encrypt). The
-configuration is ineffective on HTTP or passthrough routes.
+HSTS works only with secure routes, either edge terminated or re-encrypt. The configuration is ineffective on HTTP or passthrough routes.
```

Comment:
Some places it is spelled re-encrypt and other place reencrypt.  Whatever is used in other docs is fine.